### PR TITLE
trim down to use sanity test docs requirements file

### PR DESCRIPTION
**Pull request by samccann**
##### SUMMARY

Remove other docs requirements files as they just create problems to keep in sync with the CI requirements file.

##### ISSUE TYPE

- Docs Pull Request

##### COMPONENT NAME

docs/docsite/rst/community/documentation_contributions.rst
##### ADDITIONAL INFORMATION

```paste below

```

### CODE REVIEW

**Review by mattclay — APPROVED** [review]

File: docs/docsite/rst/community/documentation_contributions.rst [hunk 1]
```diff
@@ -84,12 +84,14 @@ To work with documentation on your local machine, you need to have python-3.5 or
     pip install --user -r requirements.txt
     pip install --user -r docs/docsite/requirements.txt
 
-The :file:`docs/docsite/requirements.txt` file allows a wide range of versions and may install new releases of required packages. New releases of these packages may cause problems with the Ansible docs build. If you want to install tested versions of these dependencies, use :file:`docs/docsite/known_good_reqs.txt` instead:
+The :file:`docs/docsite/requirements.txt` file allows a wide range of versions and may install new releases of required packages. New releases of these packages may cause problems with the Ansible docs build. If you want to install tested versions of these dependencies, use :file:`test/sanity/code-smell/docs-build.requirements.txt` instead, which matches the dependencies used by CI:
 
 .. code-block:: bash
-
+
```
Comment (felixfontein) [inline docs/docsite/rst/community/documentation_contributions.rst]: Shouldn't these spaces not be here?
```suggestion

```

File: docs/docsite/rst/community/documentation_contributions.rst [hunk 2]
```diff
@@ -84,12 +84,13 @@ To work with documentation on your local machine, you need to have python-3.5 or
     pip install --user -r requirements.txt
     pip install --user -r docs/docsite/requirements.txt
 
-The :file:`docs/docsite/requirements.txt` file allows a wide range of versions and may install new releases of required packages. New releases of these packages may cause problems with the Ansible docs build. If you want to install tested versions of these dependencies, use :file:`docs/docsite/known_good_reqs.txt` instead:
+The :file:`docs/docsite/requirements.txt` file allows a wide range of versions and may install new releases of required packages. New releases of these packages may cause problems with the Ansible docs build. If you want to install tested versions of these dependencies, use :file:`test/sanity/code-smell/docs-build.requirements.txt` instead, which matches the dependencies used by CI:
 
 .. code-block:: bash
```
Comment (samccann) [inline docs/docsite/rst/community/documentation_contributions.rst:89]:
```suggestion
.. code-block:: bash

```

**Review by ariordan-redhat — APPROVED** [review]
LGTM

**Pull request merged**